README.md

```diff
@@ -6,17 +6,14 @@
 
 
 # Introduction 
-keras2onnx enables you convert the keras models into [ONNX](https://onnx.ai).
-Initially Keras converter was developer in the project onnxmltools. To support more kinds of keras models and reduce the complexity of mixing multiple converters, keras2onnx was created to convert the keras model only.
+This Keras model converter(keras2onnx) enables you convert the keras models into [ONNX](https://onnx.ai) model format .
```

Comment:
Minor spacing and word choice suggestions for readability -- proposal sentence below:

> The keras2onnx model converter enables users to convert Keras models into the [ONNX](https://onnx.ai) model format.